generators/client-2/templates/src/main/webapp/app/layouts/navbar/navbar.html

```diff
@@ -14,14 +14,14 @@
         </div>
         <div class="navbar-collapse" [ngbCollapse]="isNavbarCollapsed" [ngSwitch]="isAuthenticated()">
             <ul class="nav navbar-nav navbar-right">
-                <li ui-sref-active="active">
-                    <a ui-sref="home" href="#/home" (click)="collapseNavbar()">
+                <li uiSrefActive="active">
+                    <a uiSref="home" (click)="collapseNavbar()">
                         <span class="glyphicon glyphicon-home"></span>
                         <span class="hidden-sm" translate="global.menu.home">Home</span>
                     </a>
                 </li>
                 <!-- jhipster-needle-add-element-to-menu - JHipster will add new menu items here -->
-                <li [ngClass]="{active: $state.includes('entity')}" *ngSwitchCase="true" ngbDropdown class="dropdown pointer">
+                <li uiSrefActive="active" *ngSwitchCase="true" ngbDropdown class="dropdown pointer">
```

Comment:
`uiSrefActive` in ng2 can enclose multiple `uiSref`.  It will add the 'active' class if any of those uiSref are active.